README.md

```diff
@@ -0,0 +1,169 @@
+2. Create bash script file: `touch setupvpn.sh`
+3. Use text editor and write the following lines to the file
+
+    ```bash
+
+    sudo tee /etc/apt/sources.list.d/pritunl.list << EOF
+    deb <http://repo.pritunl.com/stable/apt> focal main
+    EOF
+    sudo apt --assume-yes install gnupg
+    gpg --keyserver hkp://keyserver.ubuntu.com --recv-keys 7568D9BB55FF9E5287D586017AE645C0CF8E292A
+    gpg --armor --export 7568D9BB55FF9E5287D586017AE645C0CF8E292A | sudo tee /etc/apt/trusted.gpg.d/pritunl.asc
+    wget -qO - <https://www.mongodb.org/static/pgp/server-6.0.asc> | sudo apt-key add -
+    echo "deb [ arch=amd64,arm64 ] <https://repo.mongodb.org/apt/ubuntu> focal/mongodb-org/6.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-6.0.  list
+    sudo apt update
+    sudo apt install -y mongodb-org pritunl
+    sudo systemctl start mongod pritunl
+    sudo systemctl enable mongod pritunl
+
+    ```
+
+4. Execute: `sudo bash setupvpn.sh`
+
+5. After execution check service status
+    * sudo systemctl status mongod
+    * sudo systemctl status pritunl ![check status](2022-09-01-13-05-05.png)
+
+6. In your browser open refer
+    `https://bastionIPAddress/setup`
+7. Ignore warning ![warning](2022-09-01-13-07-39.png)
+8. Open bastion terminal and execute instruction from pritunl page ![command](2022-09-01-13-09-24.png)
+9. Copy-paste created key and save ![key](2022-09-01-13-12-02.png)
+10. Ignore warning
+11. Execute in bastion terminal: sudo pritunl default-password
+    * copy-paste login and password
+    * sign in ![sign in](2022-09-01-13-15-32.png)
+12. initial setup
+    * setup username (test) and password (6214157507237678334670591556762) ![change](2022-09-01-13-18-42.png)
+    * save
+
+13. Add organization and user (Usetname: test, pin: 6214157507237678334670591556762 ) ![add org and user](2022-09-01-13-21-30.png)
+    ![result](2022-09-01-13-23-24.png)
+14. Create server
+    * name - test
+    * remember port ![result](2022-09-01-13-25-35.png)
+15. Attach created organization to server and start server ![start](2022-09-01-13-27-33.png)
+
+16. Create firewall rule in VPC Network GCP use tag  `bastion` and this tag to your `bastion` VM   ![VPN](2022-09-01-13-33-02.png)
+17. Download archive contained *.ovpn ![ovpn](2022-09-01-13-38-16.png)
+18. In your localhost terminal add config `*.ovpn` to openvpn : `sudo openvpn *.ovpn`
+
+19. Open new terminal and check connection througth bastion VPN server. `sudo ssh -i ~/.ssh/appuser appuser@10.128.0.8`
+
+## TLS certificate for pritunl
+
+1. use ssh and login to bastion
+2. install snapd
+3. `sudo snap install core`
+4. `sudo snap refresh core`
+5. `sudo apt-get remove certbot`
+6. `sudo snap install --classic certbot`
+7. `sudo ln -s /snap/bin/certbot /usr/bin/certbot`
+8. `shutdown your webservice sudo systemctl stop pritunl`
+9. `sudo certbot certonly --standalone -d BastionIPAddress.nip.io`
+10. install created certificate to webaplication
+11. `sudo certbot renew --dry-run`
```

Comment:
`--dry-run` is actually do not renew your certificates (this only "simulates" renew) so this command looks excessive